### PR TITLE
create storage file share in container instance

### PR DESCRIPTION
**Pull request by wenwu449**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 101-aci-storage-file-share/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Create a Storage Account and a File Share
+
+Create a Storage Account and a File Share via az-cli in Container Instance - <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-aci-storage-file-share%2Fazuredeploy.json" target="_blank">
```
Comment (bmoore-msft) [inline 101-aci-storage-file-share/README.md]: fix formatting in the readme - buttons are on the same line as the text

File: 101-aci-storage-file-share/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-aci-storage-file-share%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template demonstrates createing Storage File Share via az-cli and Azure Container Instance, modify the template to create blob or other similar storage operations. Azure Container Instance and az-cli can be used to create services not directly available in Azure Resource Manager Template.
```
Comment (bmoore-msft) [inline 101-aci-storage-file-share/README.md]: fix typos

File: 101-aci-storage-file-share/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
++ **Storage Account**: Storage account for the file share
++ **Azure Container Instance**: Azure Container Instance, where the az-cli is exectued to create the file share
++ **File share**: [Azure File share](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction) accessiable via SMB protocol
+
+## Deployment steps
+
+You can click the "deploy to Azure" button at the beginning of this document or follow the instructions for command line deployment using the scripts in the root of this repo.
+
+## Usage
```
Comment (bmoore-msft) [inline 101-aci-storage-file-share/README.md]: remove this

File: 101-aci-storage-file-share/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            }
+        },
+        "fileShareName": {
+            "type": "string",
+            "defaultValue": "[uniquestring(resourceGroup().id)]",
+            "metadata": {
+                "description": "File Share Name"
+            }
+        },
+        "containerInstanceLocation": {
```
Comment (bmoore-msft) [inline 101-aci-storage-file-share/azuredeploy.json]: if this is only available in certain locations let's add allowed values for now - if they are not available in westus (where CI runs) then add a valid defaultValue for location

File: 101-aci-storage-file-share/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        "cpuCores": "1.0",
+        "memoryInGb": "1.5",
+        "containerGroupName":"[concat(parameters('storageAccountName'), '-createshare')]",
+        "containerName": "createshare"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[parameters('storageAccountName')]",
+            "apiVersion": "2016-01-01",
```
Comment (bmoore-msft) [inline 101-aci-storage-file-share/azuredeploy.json]: update apiVersion

File: 101-aci-storage-file-share/azuredeploy.parameters.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountType": {
```
Comment (bmoore-msft) [inline 101-aci-storage-file-share/azuredeploy.parameters.json]: remove these and just use defaultValues in the template

File: 101-aci-storage-file-share/README.md (hunk 3 above)
Comment (wenwu449) [reply 101-aci-storage-file-share/README.md]: remove the "Usage" title or the entire "Usage" part?

File: 101-aci-storage-file-share/azuredeploy.json (hunk 4 above)
Comment (wenwu449) [reply 101-aci-storage-file-share/azuredeploy.json]: added, westus is supported.

File: 101-aci-storage-file-share/README.md (hunk 3 above)
Comment (bmoore-msft) [reply 101-aci-storage-file-share/README.md]: The title - since there's nothing in that section

**Pull request merged**